ddsrouter_yaml/src/cpp/YamlReader.cpp

```diff
@@ -80,13 +81,27 @@ Yaml YamlReader::get_value_in_tag(
     }
 }
 
+template <>
+YamlReaderVersion YamlReader::get<YamlReaderVersion>(
+        const Yaml& yml,
+        const YamlReaderVersion /* Not Specialization */)
```

Comment:
Remove the `/* Not Specialization */` comment as this is no longer a template. Instead use the following:

```suggestion
        const YamlReaderVersion /* version = LATEST */)
```